trailblazer/store/base.py

```diff
@@ -80,7 +80,7 @@ def get_filtered_analyses(self, analyses: Query, query: AnalysesRequest) -> Quer
 
     def get_visible_analyses(self, analyses: Query) -> Query:
         return apply_analysis_filter(
-            filter_functions=[AnalysisFilter.FILTER_BY_IS_VISIBLE],
+            filter_functions=[AnalysisFilter.BY_IS_VISIBLE],
```

Comment:
```suggestion
            filter_functions=[AnalysisFilter.IS_VISIBLE],
```
Perhaps like this in case of bools?

Reply:
Sine we are still returning a query, I think the current naming works well. If we decide to change it should be done in a separate PR.